XCTestBootstrap/TestManager/FBTestManagerTestReporterJUnit.h

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBTestManagerTestReporter.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ A Test Reporter that implements the FBTestManagerTestReporter interface.
+ It writes the Test Result to a given File Handle in the JUnit XML format.
+ */
+@interface FBTestManagerTestReporterJUnit : NSObject <FBTestManagerTestReporter>
+
+/**
+ Constructs a JUnit Test Reporter.
+
+ @param outputFileHandle a file handle the JUnit XML should be written to.
+ */
++ (instancetype)withOutputFileHandle:(NSFileHandle *)outputFileHandle;
```

Comment:
For the `RSpec` reporter you probably want to write to `stdout` or `stderr`, I assume. I gave the `JUnit` reporter the same interface so later it will be easier to plug that into `fbsimctl`. I had an interface similar to this in mind:

```
fbsimctl --reporter=rspec # writes to stdout
fbsimctl --reporter=junit junit.xml # write to the file junit.xml
```

If all reporters share the same interface (init with a file handle), it will be easier. But on the other hand you will probably never want to write the `JUnit` output to `stdout` or `stderr`, but always to a file. I guess I should change the interface then.